pkg/apigw/profiler/hit.go

```diff
@@ -0,0 +1,57 @@
+package profiler
+
+import (
+	"encoding/base64"
+	"fmt"
+	"time"
+
+	h "github.com/cortezaproject/corteza-server/pkg/http"
+)
+
+type (
+	Hit struct {
```

Comment:
what was the rationale for abbreviating fields in this struct?

Reply:
None other than self-explanatory